### PR TITLE
Update PPMC -> PMC

### PR DESCRIPTION
**Pull request by ulysses-you**

### CODE REVIEW

**Comment by ulysses-you** [comment]
@yaooqinn

**Comment by yaooqinn** [comment]
let's keep the news unchanged

**Comment by ulysses-you** [comment]
@yaooqinn addressed

File: src/en/become_pmc_member.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ menu:
 ## Become A Podling PMC member of Apache Kyuubi
```
Comment (yaooqinn) [inline src/en/become_pmc_member.md:71]: Become A ~~Podling~~ PMC member of Apache Kyuubi

File: src/en/become_pmc_member.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ menu:
...
 
 Anyone being supportive of the community and working in any of the
-CoPDoC areas can become an Apache Kyuubi Podling PMC(PPMC) member.
```
Comment (yaooqinn) [inline src/en/become_pmc_member.md:74]: Apache Kyuubi PMC member.

**Comment by yaooqinn** [comment]
Can you search the word `Podling` from the project and remove it if necessary?

**Comment by ulysses-you** [comment]
@yaooqinn  addressed

**Review by yaooqinn — APPROVED** [review]

**Pull request merged**